recipes/cdp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cdp" %}
+{% set version = "1.3.3" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CDAT/cdp/archive/v{{ version }}.tar.gz
+  sha256: 8c3e69c695ecdf0c8e3d0fe38e4ae6987445aff104a89e837e496818ff9bb343
+
+build:
+    number: 0
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python {{ python }}
+    - setuptools
+    - pip
+  run:
+    - python {{ python }}
+    - output_viewer
+    - pyyaml
+    - dask
+    - distributed
+    - six
+
+test:
+  commands:
+      - python -c "import cdp"  
```

Comment:
This can be an `import` test:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-imports

and test the command line tool that goes along with the `entry_point`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands